deploy/kubernetes/helm/che/templates/configmap.yaml

```diff
@@ -70,7 +70,14 @@ data:
   CHE_INFRA_KUBERNETES_INGRESS_ANNOTATIONS__JSON: '{"kubernetes.io/ingress.class": "nginx", "{{ .Values.global.ingressAnnotationsPrefix }}ingress.kubernetes.io/rewrite-target": "/","{{ .Values.global.ingressAnnotationsPrefix }}ingress.kubernetes.io/ssl-redirect": "false","{{ .Values.global.ingressAnnotationsPrefix }}ingress.kubernetes.io/proxy-connect-timeout": "3600","{{ .Values.global.ingressAnnotationsPrefix }}ingress.kubernetes.io/proxy-read-timeout": "3600"}'
 {{- end }}
   CHE_INFRA_KUBERNETES_SERVER__STRATEGY: {{ .Values.global.serverStrategy }}
+{{- if .Values.global.log.loggerConfig }}
+  CHE_LOGGER_CONFIG: {{ .Values.global.log.loggerConfig }}
+{{- end }}
+{{- if .Values.global.log.customAppenderName }}
+  CHE_LOGS_APPENDERS_IMPL: {{ .Values.global.log.customAppenderName }}
+{{- else }}
```

Comment:
Optional: Consider setting `plaintext` appender as default value and removing `else` clause to simplify file

Reply:
Good suggestion. Done.